<?xml version="1.0"?>
<launch>

    <!-- Fake imaging node -->
    <node pkg="hrii_task_board_fsm" type="imaging_node" name="imagin_node" output="screen"/>
     
</launch>